arangod/StorageEngine/RocksDBOptionFeature.cpp

```diff
@@ -164,6 +166,15 @@ void RocksDBOptionFeature::collectOptions(std::shared_ptr<ProgramOptions> option
                      "You can increase the amount only",
                      new UInt64Parameter(&_maxWriteBufferNumber));
 
+  options->addOption("--rocksdb.max-write-buffer-number-to-maintain",
+                     "maximum number of immutable write buffers that build up in memory "
```

Comment:
Should we specify the unit here ( I assume MB?)

Reply:
This is unitltest, it is actually a number of write buffers.